service-search-opendata/src/main/java/fi/nls/oskari/search/OpenStreetMapSearchChannel.java

```diff
@@ -130,9 +158,41 @@ public ChannelSearchResult doSearch(SearchCriteria searchCriteria) {
                 }
                 searchResultList.addItem(item);
             }
-        } catch (Exception e) {
-            log.error(e, "Failed to search locations from register of OpenStreetMap");
+        } catch(Exception e) {
+            log.error(e, "Failed to convert JSONArray to ChannelSearchResult");
         }
+
         return searchResultList;
     }
+    @Override
+    public ChannelSearchResult reverseGeocode(SearchCriteria searchCriteria) throws IllegalSearchCriteriaException {
+        try {
+            String srs = searchCriteria.getSRS();
+            if( srs == null ) {
+                srs = "EPSG:3067";
```

Comment:
Lets use this for default: `PropertyUtil.get("oskari.native.srs", "EPSG:3857");`